cw-orch/src/daemon/sender.rs

```diff
@@ -125,7 +127,8 @@ impl Sender<All> {
         );
 
         let auth_info =
-            SignerInfo::single_direct(Some(self.private_key.public_key()), sequence).auth_info(fee);
+            SignerInfo::single_direct(Some(self.cosmos_private_key().public_key()), sequence)
```

Comment:
Wow so the public key is derived from the cosmos private key but the signing requires a different protocol? That's interesting.

Reply:
Ah, no didn't notice we also have to change things here